lib/tasks/move_profile.rake

```diff
@@ -1,8 +1,24 @@
-namespace :move_profile do
-  desc 'backfill move profile ids'
-  task backfill: :environment do
-    Move.includes(person: :profiles).find_each.reject { |m| m.profile_id.present? }.each do |move|
-      move.update!(profile_id: move.person.latest_profile.id)
+namespace :moves do
+  desc 'Set all profile ids in all the moves'
+  task set_profiles: :environment do
+    moves = Move.where(profile_id: nil).includes(person: :profiles)
+
+    total = moves.count
+
+    if moves.empty?
+      puts 'All profiles IDs were already updated.'
+      return
     end
+
+    moves.find_each(batch_size: 200).with_index do |move, n|
+      puts "#{n}/#{total} moves processed ..." if (n % 200).zero? # Show progression
```

Comment:
Reminds me of https://github.com/paul/progress_bar :). Not suggesting we add it. But have used in the past.

Reply:
:) it is just temporary, after a while we are going to remove it